src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,905 @@
+
+(display: "The Gutter - Bathroom - Menu")
+
+::The Gutter - Bathroom - Sink puke
+I spewed all over the floor, splashing on my feet, before I finally found the sink. I nearly filled it up with whatever needed out of me.
+
+When I was finished I was gasping, like I’d been punched in the gut. This night was all too much. Anna gone, maybe dead already. I’ve been drugged, chased, and now I’ve gotta ask my questions with puke on my shoes.
+
+I wiped my mouth with the back of my hand. When this was over, I was gonna need a high the length of I-55 to deal.
+
+(display: "The Gutter - Bathroom - Menu")
+
+(set: $sinkPuke to true)
+
+::The Gutter - Bathroom - Menu
+>>[[I headed back out into the din and dipshits|The Gutter - Inside hub]]
+(if: $mirrorSeen is false)[>>[[I was drawn to my reflection in the mirror|The Gutter - Bathroom - Mirror]].]
+(if: $toiletflashbackSeen is false)[>>[[I sat on the toilet to catch my breath|The Gutter - Bathroom - Toilet flashback]].]
+
+::The Gutter - Bathroom - Mirror
+(set: $mirrorSeen to true)
+I gripped the edges of the counter a bit tighter. My face looked unfamiliar. The stranger’s eyes were weary, like they’d been hung with weights.
+
+//“Hey. $playerName," the mirror stranger said, "You look pretty fucked up. Like somebody dosed you.”//
+
+(click: "dosed")[$dissolve["Yeah, well. How about we skip the obvious and you give me a goddamn hand here?"]]
+
+(click: "give me a goddamn hand")[$dissolve[//“A hand? HA! That’s richer than the fuckos in those nice trailers,"// The face got even nastier, contorting and screaming at me like a high wind.
+
+//"How about you stop sobbing in the potty and find your friend? Are you gonna crumble this time you no-good bitch? You fucking idiot! How are you gonna handle when it all comes down? When the rubber hits the road?”//]]
+
+(click: "rubber hits the road")[$dissolve[My jaw slacked and my lip quivered. I held my hand up and watched it tremble for a moment. But then, like Jay-sus settling the bubbles in a shook up bottle of Rock N Rye, my muscles went calm.]]
+
+(click: "calm")[$dissolve[I fixed myself into a battle stance and roared back, “I handle fine. I handle better! I ride steady and smooth like a ‘65 Cutlass at midnight with the headlights off and I’m COMING]]
+
+(click: "COMING")[THROUGH]
+
+(click: "THROUGH")[YOUR]
+
+(click: "YOUR")[FUCKING]
+
+(click: "FUCKING")[B U L L S H I T]
+
+(click: "B U L L S H I T")[$dissolve[I heard a crunching at the edge of my perception.]]
+
+(click: "crunching")[$dissolve[I looked down to see the sink starting to shift out of the wall. I had to scramble to keep it in place.(if: $sinkPuke is true)[ A bit of my puke sloshed onto my hand from the sink. Ugh.] I hadn’t realized how hard I’d been gripping it.]]
+
+(click: "gripping it")[$dissolve[The mirror stranger got me riled. She sure was a bitch, but she’s dead on. Funny how she’s fucking right as can be.
+
+(display: "The Gutter - Bathroom - Menu")]]
```

Comment:
This is something that exists in pretty much every list of options I've built. Not sure how to close that space when an option disappears. Nick noted it in the Mom's Tent code as well. So, he might have an idea. I could probably fix it given time, but it would be low priority for me to teach myself. This is definitely in the zone of things that usually takes a couple of hours for me to test and learn. None of the quick fixes I tried while drafting seemed to work.

Reply:
I've noted this in other instances here, and I'm happy to walk through the easiest fix I've found today. But in short:
- end the preceding line with a `\`
- After your `(if: $variable is true/false)[`, immediately insert a new line
- Repeat as needed for any other conditional options. Basically, this just ensures that a new line is _only_ inserted when the `(if:)` condition is satisfied.